### PR TITLE
Ntassone/container example

### PR DESCRIPTION
**Pull request by ntassone**
added custom container example page

### CODE REVIEW

File: examples/container.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    </script>
+
+    <div id="common"></div>
+
+    <div id="expected">
+      <p>
+      The expected behaviour of this page is as follows:
+      </p>
+      <ul>
+        <li>The UserList should appear on the <strong>right</strong> of the screen.</li>
```
Comment (nealstewart) [inline examples/container.html]: The explanation of the page should be updated.

File: examples/container.html [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+        <li>The UserList should <strong>not</strong> be initially collapsed.</li>
+        <li>A new user should be appended each time this example is loaded in a new browser.</li>
+        <li>Usernames should be truncated after <strong>10</strong> characters.</li>
+        <li>Upon clicking the <strong>Give me an avatar!</strong> button, every user list should update that user with an avatar.</li>
+        <li>Changing your name via the UI should update <strong>your name</strong> in the user list for all users.</li>
+        <li>To submit your name change you can click the arrow, press enter, or tab out of the input.</li>
+      </ul>
+    </div>
+
+    <button id="avatar">Give me an avatar!</button>
```
Comment (nealstewart) [inline examples/container.html]: Do you want this functionality on this page?

File: examples/container.html [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+        var userList = new UserList(options);
+
+        userList.initialize(function(err) {
+          if (err) {
+            throw err;
+          }
+        });
+
+        document.getElementById('avatar').onclick = function() {
```
Comment (nealstewart) [inline examples/container.html]: You probably want to remove this.

**Comment by nealstewart** [comment]
LGTM pending :white_check_mark:

**Comment by ntassone** [comment]
Test failure is unrelated to this change

**Pull request merged**